02-Context-or-Narrative/State-of-PureScript-Documentation-2019.md

```diff
@@ -1,28 +1,37 @@
 # State of PureScript Documentation: 2019
 
+<!-- https://magnetikonline.github.io/markdown-toc-generate/ -->
+
+- [Why Read This Document?](#why-read-this-document)
+- [What is "Good" Documentation Anyway?](#what-is-good-documentation-anyway)
+    - [The Types of Documentation](#the-types-of-documentation)
+    - [The Documentation's Intended Audience](#the-documentations-intended-audience)
+    - [Maintaining Documentation's Accuracy](#maintaining-documentations-accuracy)
+        - [The "Size" of a Change](#the-size-of-a-change)
+        - [The "Frequency" of a Change](#the-frequency-of-a-change)
+        - [How to Make Maintenance Easier](#how-to-make-maintenance-easier)
+    - [Criteria for "Good" Documentation](#criteria-for-good-documentation)
+    - [Evaluating PureScript's Documentation](#evaluating-purescripts-documentation)
+- [Why is PureScript's Documentation Lacking and How Do We Improve It?](#why-is-purescripts-documentation-lacking-and-how-do-we-improve-it)
+- [New Learners: What is the Best Way to Learn PureScript?](#new-learners-what-is-the-best-way-to-learn-purescript)
+- [PureScript Documentation Writers: What is the Best Way to Write Documentation in this Context?](#purescript-documentation-writers-what-is-the-best-way-to-write-documentation-in-this-context)
+- [Core Contributors: ???](#core-contributors-)
+
+
 ## Why Read This Document?
 
-This document is not meant to convince you to use PureScript. We assume that readers already believe in using PureScript instead of alternatives.
+This document is not meant to convince you to use PureScript. We assume that readers already want to learn to use PureScript.
```

Comment:
'We assume that readers are already interested in PureScript'. I think this would generalize the intended meaning well enough.

Reply:
Mm... that sounds better. Still, I think it could be broadened slightly to include more of the notion of users besides those who are just interested. How about this rendering?
> We assume that readers are already interested in or use PureScript.